requirements/base.in

```diff
@@ -6,6 +6,7 @@ drf-jwt
 django-waffle
 edx-django-utils>=3.8.0      # using new set_custom_attribute method
 edx-opaque-keys
+edx-toggles
```

Comment:
**Note to Self:** Remove this. Instead of using a `SettingToggle`, I just used a Django Setting directly.

Reply:
Delete this line:
```suggestion
```